src/xdocs/google_style.xml

```diff
@@ -799,17 +799,21 @@
                 <td>
                   <span class="wrapper inline">
                     <img
-                        src="images/ok_blue.png"
```

Comment:
I don't think we need to change it to green because we don't cover all Google requirements.

Reply:
okayy I will revert this 👍

Reply:
done